<ui version="4.0" stdsetdef="1">
  <author>Peter Bartfai</author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>MpdModelSelectionPanel</class>
  <widget class="QDialog" name="MpdModelSelectionPanel">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>300</width>
        <height>200</height>
      </rect>
    </property>
    <property name="sizePolicy">
      <sizepolicy>
        <hsizetype>5</hsizetype>
        <vsizetype>5</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
      </sizepolicy>
    </property>
    <property name="minimumSize">
      <size>
        <width>300</width>
        <height>200</height>
      </size>
    </property>
    <property name="windowIcon">
      <iconset resource="resources.qrc" >
      <normaloff>:/images/images/LDViewIcon16.png</normaloff>:/images/images/LDViewIcon16.png</iconset>
    </property>
    <property name="baseSize">
      <size>
        <width>300</width>
        <height>200</height>
      </size>
    </property>
    <property name="windowTitle">
      <string>LDView MPD Model Selection</string>
    </property>
    <property name="sizeGripEnabled">
      <bool>true</bool>
    </property>
    <layout class="QVBoxLayout">
      <property name="margin">
        <number>11</number>
      </property>
      <property name="spacing">
        <number>4</number>
      </property>
      <item>
        <widget class="QLabel" name="selectLabel">
          <property name="text">
            <string>Select the model to display</string>
          </property>
          <property name="alignment">
            <set>Qt::AlignVCenter|Qt::AlignLeft</set>
          </property>
          <property name="wordWrap">
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QListWidget" name="modelList"/>
      </item>
      <item>
        <layout class="QHBoxLayout">
          <property name="margin">
            <number>11</number>
          </property>
          <property name="spacing">
            <number>4</number>
          </property>
          <item>
            <spacer name="Spacer3">
              <property name="sizeHint">
                <size>
                  <width>20</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="okButton">
              <property name="sizePolicy">
                <sizepolicy>
                  <hsizetype>5</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text">
                <string>OK</string>
              </property>
              <property name="shortcut">
                <string/>
              </property>
              <property name="default">
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="cancelButton">
              <property name="sizePolicy">
                <sizepolicy>
                  <hsizetype>0</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text">
                <string>Cancel</string>
              </property>
              <property name="shortcut">
                <string>Esc</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="4" margin="11"/>
  <pixmapfunction></pixmapfunction>
  <resources>
   <include location="resources.qrc" />
  </resources>
</ui>
